<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ADC7K</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QGroupBox" name="boardInfoGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Board Info</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolBox" name="boardInfoToolBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="generalPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>242</width>
                <height>382</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="label">
               <string>General</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout_8">
               <item row="0" column="0">
                <layout class="QFormLayout" name="generalFormLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="typeLabel">
                   <property name="text">
                    <string>Type</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="nameLabel">
                   <property name="text">
                    <string>Name</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="samplingRateLabel">
                   <property name="text">
                    <string>Sampling Rate</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="typeValueLabel">
                   <property name="text">
                    <string>unknown</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="nameValueLabel">
                   <property name="text">
                    <string>unknown</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLabel" name="samplingRateValueLabel">
                   <property name="text">
                    <string>unknown</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="samplerLengthLabel">
                   <property name="text">
                    <string>Sampler Length</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QSpinBox" name="samplerLengthSpinBox">
                   <property name="specialValueText">
                    <string>unknown</string>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>4194304</number>
                   </property>
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="samplerDividerLabel">
                   <property name="text">
                    <string>SamplerDivider</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QSpinBox" name="samplerDividerSpinBox">
                   <property name="maximum">
                    <number>255</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="registersPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>242</width>
                <height>382</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Registers</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <layout class="QFormLayout" name="registersFormLayout">
                 <property name="fieldGrowthPolicy">
                  <enum>QFormLayout::FieldsStayAtSizeHint</enum>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="registerAddressLabel">
                   <property name="text">
                    <string>Address</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="registerValueLabel">
                   <property name="text">
                    <string>Value</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QSpinBox" name="registerAddressSpinBox">
                   <property name="maximum">
                    <number>31</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QPushButton" name="registerReadPushButton">
                   <property name="text">
                    <string>Read</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QPushButton" name="registerWritePushButton">
                   <property name="text">
                    <string>Write</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="registerValueLineEdit">
                   <property name="maximumSize">
                    <size>
                     <width>96</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>0x00000000</string>
                   </property>
                   <property name="maxLength">
                    <number>10</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="adcPage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>242</width>
                <height>382</height>
               </rect>
              </property>
              <attribute name="label">
               <string>ADC</string>
              </attribute>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="1" column="0">
                <layout class="QFormLayout" name="adcFormLayout">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="adcResetPushButton">
                   <property name="text">
                    <string>Reset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="adcAddressLabel">
                   <property name="text">
                    <string>Address</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="adcDataLabel">
                   <property name="text">
                    <string>Data</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QPushButton" name="adcWritePushButton">
                   <property name="text">
                    <string>Write</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="adcAddressSpinBox">
                   <property name="suffix">
                    <string/>
                   </property>
                   <property name="prefix">
                    <string>0x</string>
                   </property>
                   <property name="maximum">
                    <number>255</number>
                   </property>
                   <property name="value">
                    <number>37</number>
                   </property>
                   <property name="displayIntegerBase">
                    <number>16</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QSpinBox" name="adcDataSpinBox">
                   <property name="prefix">
                    <string>0x</string>
                   </property>
                   <property name="maximum">
                    <number>65535</number>
                   </property>
                   <property name="value">
                    <number>64</number>
                   </property>
                   <property name="displayIntegerBase">
                    <number>16</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Adc7kDisplay" name="adc7kDisplayWidget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionBoardInfo"/>
    <addaction name="actionFull_Screen"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionBoardInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Board Info</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Adc7kDisplay</class>
   <extends>QWidget</extends>
   <header location="global">adc7kdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
